api/web3server.go

```diff
@@ -166,128 +166,129 @@ func (svr *Web3Server) handlePOSTReq(req *http.Request) interface{} {
+	case "eth_chainId":
+		res, err = svr.getChainID()
+	case "eth_blockNumber":
+		res, err = svr.getBlockNumber()
+	case "eth_getBalance":
+		res, err = svr.getBalance(params)
+	case "eth_getTransactionCount":
+		res, err = svr.getTransactionCount(params)
+	case "eth_call":
+		res, err = svr.call(params)
+	case "eth_getCode":
+		res, err = svr.getCode(params)
+	case "eth_protocolVersion":
+		res, err = svr.getProtocolVersion()
+	case "web3_clientVersion":
+		res, err = svr.getNodeInfo()
+	case "net_version":
+		res, err = svr.getNetworkID()
+	case "net_peerCount":
+		res, err = svr.getPeerCount()
+	case "net_listening":
+		res, err = svr.isListening()
+	case "eth_syncing":
+		res, err = svr.isSyncing()
+	case "eth_mining":
+		res, err = svr.isMining()
+	case "eth_hashrate":
+		res, err = svr.getHashrate()
+	case "eth_getLogs":
+		var filter *filterObject
+		if filter, err = parseLogRequest(web3Req.Get("params")); err == nil {
+			res, err = svr.getLogs(filter)
+		}
+	case "eth_getBlockTransactionCountByHash":
+		res, err = svr.getBlockTransactionCountByHash(params)
+	case "eth_getBlockByNumber":
+		res, err = svr.getBlockByNumber(params)
+	case "eth_estimateGas":
+		res, err = svr.estimateGas(params)
+	case "eth_sendRawTransaction":
+		res, err = svr.sendRawTransaction(params)
+	case "eth_getTransactionByHash":
+		res, err = svr.getTransactionByHash(params)
+	case "eth_getTransactionByBlockNumberAndIndex":
+		res, err = svr.getTransactionByBlockNumberAndIndex(params)
+	case "eth_getTransactionByBlockHashAndIndex":
+		res, err = svr.getTransactionByBlockHashAndIndex(params)
+	case "eth_getBlockTransactionCountByNumber":
+		res, err = svr.getBlockTransactionCountByNumber(params)
+	case "eth_getTransactionReceipt":
+		res, err = svr.getTransactionReceipt(params)
+	case "eth_getStorageAt":
+		res, err = svr.getStorageAt(params)
+	case "eth_getFilterLogs":
+		res, err = svr.getFilterLogs(params)
+	case "eth_getFilterChanges":
+		res, err = svr.getFilterChanges(params)
+	case "eth_uninstallFilter":
+		res, err = svr.uninstallFilter(params)
+	case "eth_newFilter":
+		var filter *filterObject
+		if filter, err = parseLogRequest(web3Req.Get("params")); err == nil {
+			res, err = svr.newFilter(filter)
+		}
+	case "eth_newBlockFilter":
+		res, err = svr.newBlockFilter()
+	case "eth_coinbase", "eth_getUncleCountByBlockHash", "eth_getUncleCountByBlockNumber", "eth_sign", "eth_signTransaction", "eth_sendTransaction", "eth_getUncleByBlockHashAndIndex", "eth_getUncleByBlockNumberAndIndex", "eth_pendingTransactions":
+		res, err = svr.unimplemented()
+	default:
+		res, err = nil, errors.Wrapf(errors.New("web3 method not found"), "method: %s\n", web3Req.Get("method"))
 	}
-	return web3Resps
+	if err != nil {
+		log.L().Error("web3server",
+			zap.String("requestParams", fmt.Sprintf("%+v", web3Req)),
+			zap.Error(err))
+	}
+	web3ServerMtc.WithLabelValues(method.(string)).Inc()
+	web3ServerMtc.WithLabelValues("requests_total").Inc()
```

Comment:
what's the definition of "requests_total"?

Reply:
Subrequests are included. For example, #requests of [{A, B}] is 2